app/src/androidTest/java/com/sdp/movemeet/utility/LocationFetcherTest.java

```diff
@@ -51,7 +51,7 @@
             FragmentTestRule.create(MainMapFragment.class);
 
     @Rule
-    public GrantPermissionRule mRuntimePermissionRule = GrantPermissionRule.grant(Manifest.permission.ACCESS_FINE_LOCATION);
+    public GrantPermissionRule locationRule = GrantPermissionRule.grant(Manifest.permission.ACCESS_FINE_LOCATION);
```

Comment:
Yes, variable naming convention. It's surprising, even if we refactored those names a few weeks ago, we always find variable names that are still unchanged! 😂

Reply:
indeed haha